examples/httpserver_simplepolling.py

```diff
@@ -30,6 +30,7 @@ def base(request):  # pylint: disable=unused-argument
 
 while True:
     try:
+        # do something useful in this section
```

Comment:
It make be helpful to give an example of what you mean by "something useful", like "operate hardware" or something.

Reply:
Should be easy to add some more details to that comment.